src/get-started/java-binding.md

```diff
@@ -395,6 +466,62 @@ at `/api/services/cryptocurrency/balance/:walletId`.
 See [documentation][vertx-web-docs] on the possibilities of `Vert.x` used as a web
 framework.
 
+### Configuration
+
+#### Initialization and Initial Configuration
+
+If a service needs to initialize its persistent state when it is started,
+potentially, using some configuration parameters, it shall
```

Comment:
Nit: I'm not sure I understand this usage of "shall"; looks posh. I'd say something like "Service can initialize its persistent state on its start using `Service#initialize`".